x/claim/client/cli/tx.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+)
+
+var (
+	DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
+)
+
+// nolint
+const (
+	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
```

Comment:
these two has not been used anywhere

Reply:
[e735493](https://github.com/arkeonetwork/arkeo/pull/29/commits/e735493ed751af9404e8c9177d4bc04de755a1b2)